### PR TITLE
Add enemies

### PR DESCRIPTION
**Pull request by mikegreber**

### CODE REVIEW

File: Descension/Assets/Prefab/Pickups/TorchItemPrefab.prefab [hunk 1]
```diff
@@ -177,7 +177,7 @@ BoxCollider2D:
   m_Enabled: 1
   m_Density: 1
   m_Material: {fileID: 0}
-  m_IsTrigger: 0
+  m_IsTrigger: 1
```
Comment (jsmith465264) [inline Descension/Assets/Prefab/Pickups/TorchItemPrefab.prefab:180]: maybe need to undo these changes?

**Review by reillykeele — APPROVED** [review]
